apps/marketplace/pages/PublishAnswerPage.js

```diff
@@ -0,0 +1,191 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { Form, actions } from 'react-redux-form'
+import { withRouter } from 'react-router-dom'
+import DocumentTitle from 'react-document-title'
+import AUpageAlert from '@gov.au/page-alerts/lib/js/react.js'
+import AUheading from '@gov.au/headings/lib/js/react.js'
+import format from 'date-fns/format'
+import { publishAnswer } from 'marketplace/actions/briefActions'
+import { loadQuestion } from 'marketplace/actions/questionActions'
+import { required } from 'marketplace/components/validators'
+import Textarea from 'shared/form/Textarea'
+import { rootPath } from 'marketplace/routes'
+import LoadingIndicatorFullPage from 'shared/LoadingIndicatorFullPage/LoadingIndicatorFullPage'
+
+const model = 'publishAQuestionForm'
+
+class PublishAnswerPage extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      loading: false,
+      errorMessage: '',
+      saved: false,
+      brief: null,
+      question: null
+    }
+  }
+
+  componentWillMount() {
+    this.setState({
+      loading: true
+    })
+    const briefId = this.props.match.params.briefId
+    const questionId = this.props.match.params.questionId
+    if (briefId.length > 0) {
```

Comment:
Do we need the check on length or the value of `briefId`?

Reply:
updated